src/backEndClient.ts

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+const getExample = axios.get("/api/Example");
+
+export default getExample;
```

Comment:
You can only have one `default` export per module and in the future, we'll add more functions to this file, so specifying one `default` function doesn't make a lot of sense here.

I think you can specify a non default export  with `export {getExample}` and then we'll put other functions when we have them: `export {getExample, getOtherExample, postSomethingEtc}`

Reply:
I started doing another related task, and figured out one can do `export default { getExample, .... }`